README.md

```diff
@@ -8,7 +8,7 @@ A Java API for libaio.
 ```bash
 mkdir Release
 cd Release
-cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+cmake -DCMAKE_BUILD_TYPE=RELEASE ../libjaio/
```

Comment:
The intention was that you create the Release directory in libjaio.
i.e. I would add a ``cd libjaio`` at the beginning